azavea/echo-locator
# Bugfix/kak/update craisglist links#266

## Overview

Two-character change to fix Craigslist links.

Also update dependencies.


## Testing Instructions

 * Reinstall packages via `yarn`
 * Click the Craigslist link in a neighborhood detail sidebar
 * Search should return results (if there are any for the given terms)
 
Fixes #266
